[replacing D41588356] Trying to fix issues in TP serialization (#89765)

pls331 opened the pull request:
Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/89765

Differential Revision:
D41588356

LaMa Project: L1142320

houseroad commented on torch/package/package_exporter.py:
```diff
@@ -85,7 +85,9 @@ def __repr__(self):
     SOURCE_FILE_NOT_FOUND = (
-        "Module had a __file__, but we could not find it in your filesystem."
+        "Module had a __file__, but we could not find it in your filesystem. "
+        + "You many want to add this file into extern/intern/mock for your "
```
You may

houseroad reviewed (approved):
Looks good. Please fix the typo.

pls331 committed:
[replacing D41588356] Trying to fix issues in TP serialization (#89765) (#89765)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/89765

LaMa Project: L1142320

Pull Request resolved: https://github.com/pytorch/pytorch/pull/89861

Reviewed By: houseroad

Differential Revision: D41588356

Pulled By: RoshanPAN

fbshipit-source-id: 88f65176fd3acc4ed4494b166bc021ebd7da66be
